ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -48,8 +48,8 @@ class BottomCommandingDemoController: DemoController {
         return Array(1...25).map {
             let item = CommandingItem(title: "Item " + String($0), image: homeImage, action: commandAction)
             item.selectedImage = homeSelectedImage
-            item.isOn = $0 % 3 == 1 ? true : false
```

Comment:
Did you update your after screenshots after making this change? I changed it away from just changing `modifiedCommandIndices` so that we could more easily see all the combination of on/off enabled/disabled, and also so that it would change the commands in the overflow section.

Reply:
I did not update the screenshot. I think the use of the modulo makes sense to achieve that behavior so I've added the code back.